lte/gateway/c/session_manager/SetMessageManagerHandler.cpp

```diff
@@ -86,22 +94,24 @@ void SetMessageManagerHandler::SetAmfSessionContext(
      */
     // Requested message from AMF to release the session
     if (cfg.common_context.sm_session_state() == RELEASED_4) {
-      if (cfg.common_context.sm_session_version() != 0) {
-        MLOG(MERROR) << "Wrong version received from AMF for IMSI " << imsi
-                     << " but continuing release request";
+      if (cfg.common_context.sm_session_version() == 0) {
+        MLOG(MERROR) << "Wrong version received from AMF for IMSI " << imsi;

```

Comment:
From my experience I've often found it's too ambiguous to only log IMSI. Should we log pdu_id/something else as well here? Also a nit is that the imsi value is already prefixed with the string 'IMSI', so lets omit it here. 
```
MLOG(MERROR) << "Wrong version received from AMF for " << imsi << " otherID " << <otherID>;
```

Reply:
Added pdu_id